public/game/actions.js

```diff
@@ -117,13 +118,26 @@ function addCardToHand(state, {card}) {
 }
 
 // Discard a single card from your hand.
-function discardCard(state, {card}) {
+function discardCard(state, {card}, exhaust) {
```

Comment:
Since `exhaust` is already defined on `card.exhaust` we don't need to pass it around as the third argument here.